File: app/src/main/java/com/soumya/wwdablu/zomatobuddy/model/search/Location.java
package com.soumya.wwdablu.zomatobuddy.model.search;

public class Location{
	private String zipcode;
	private String address;
	private String city;
	private String localityVerbose;
	private String latitude;
	private String locality;
	private int countryId;
	private int cityId;
	private String longitude;

	public void setZipcode(String zipcode){
		this.zipcode = zipcode;
	}

	public String getZipcode(){
		return zipcode;
	}

	public void setAddress(String address){
		this.address = address;
	}

	public String getAddress(){
		return address;
	}

	public void setCity(String city){
		this.city = city;
	}

	public String getCity(){
		return city;
	}

	public void setLocalityVerbose(String localityVerbose){
		this.localityVerbose = localityVerbose;
	}

	public String getLocalityVerbose(){
		return localityVerbose;
	}

	public void setLatitude(String latitude){
		this.latitude = latitude;
	}

	public String getLatitude(){
		return latitude;
	}

	public void setLocality(String locality){
		this.locality = locality;
	}

	public String getLocality(){
		return locality;
	}

	public void setCountryId(int countryId){
		this.countryId = countryId;
	}

	public int getCountryId(){
		return countryId;
	}

	public void setCityId(int cityId){
		this.cityId = cityId;
	}

	public int getCityId(){
		return cityId;
	}

	public void setLongitude(String longitude){
		this.longitude = longitude;
	}

	public String getLongitude(){
		return longitude;
	}
}

File: app/src/main/java/com/soumya/wwdablu/hungry/network/model/search/Location.kt
package com.soumya.wwdablu.hungry.network.model.search

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class Location(
        var address: String,
        var locality: String,
        var city: String,

        @SerializedName("city_id")
        var cityId: Int,

        var latitude: String,
        var longitude: String,
        var zipcode: String,

        @SerializedName("country_id")
        var countryId: Int,

        @SerializedName("locality_verbose")
        var localityVerbose: String

) : Parcelable